src/prowjobsscraper/equinix_metadata.py

```diff
@@ -47,6 +48,11 @@ def _set_equinix_metadata(self, job: ProwJob) -> None:
         except exceptions.ClientError as e:
             logger.debug("Equinix metadata are missing from %s: %s", metadata_path, e)
 
+        except auth_exceptions.InvalidOperation as e:
```

Comment:
before patching it let's first identify what URI we're unable to access
maybe even at first stage do:
```
except auth_exceptions.InvalidOperation as e:
    raise RuntimeError(f"Forbidden access to {metadata_path}") from e
```